main.tf

```diff
@@ -89,34 +95,45 @@ module "database" {
 module "user_data" {
   source = "./modules/user_data"
 
-  tfe_license_secret     = var.tfe_license_secret
   active_active          = local.active_active
-  aws_access_key_id      = var.aws_access_key_id
-  aws_bucket_data        = module.object_storage.s3_bucket.id
-  aws_region             = data.aws_region.current.name
-  aws_secret_access_key  = var.aws_secret_access_key
+  tfe_license_secret     = var.tfe_license_secret
+  enable_external        = local.enable_external
+  airgap_url             = var.airgap_url
   fqdn                   = local.fqdn
   iact_subnet_list       = var.iact_subnet_list
   iact_subnet_time_limit = var.iact_subnet_time_limit
   kms_key_arn            = module.kms.key.arn
   ca_certificate_secret  = var.ca_certificate_secret
 
+  # mounted disk
+  enable_disk      = local.enable_disk
+  disk_device_name = local.disk_device_name
```

Comment:
`local.disk_device_name` is not declared.

Reply:
removed unused var. It was only used while i was testing.
tested by creating https://cell.tfe-modules-test.aws.ptfedev.com:8800

Reply:
Note for posterity: Here in the [GCP tfe_init script](https://github.com/hashicorp/terraform-google-terraform-enterprise/blob/main/modules/user_data/templates/tfe_vm.sh.tpl#L111-L134), the script does some checking for the mounted disk, and I don't see that here or in the AzureRM module, therefore I added an [Asana task](https://app.asana.com/0/1200681690870921/1201789596800947) to handle this in the [tfe_init module](https://github.com/hashicorp/terraform-random-tfe-utility/tree/main/modules/tfe_init).